python/thunder/imgprocessing/registration.py

```diff
@@ -138,8 +139,11 @@ def fitandtransform(im, reg):
         return Images(newrdd).__finalize__(images)
 
 
-class RegistrationModel(object):
-
+class RegistrationModel(Serializable, object):
```

Comment:
See comments on the next 4 lines. :) Basically, you can't put object first, much as one might want to. See for instance: http://stackoverflow.com/questions/3003053/metaclass-multiple-inheritance-inconsistency